Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -165,6 +178,32 @@ void DMainWindow::OnOpen()
 		StartGame(filename);
 }
 
+void DMainWindow::OnBrowse()
+{
+	std::string path = ShowFolderDialog().toStdString();
+	if (!path.empty()){
+		std::vector<std::string>::iterator itResult = std::find(
+				SConfig::GetInstance().m_ISOFolder.begin(),
+				SConfig::GetInstance().m_ISOFolder.end(), path);
+
+		if (itResult == SConfig::GetInstance().m_ISOFolder.end())
+		{
+			SConfig::GetInstance().m_ISOFolder.push_back(path);
+			SConfig::GetInstance().SaveSettings();
+		}
+	}
+	m_game_tracker->ScanForGames();
+}
+
+void DMainWindow::OnExit()
+{
+	this->close();
+	if (Core::GetState() == Core::CORE_UNINITIALIZED || m_isStopping)
+		return; // already stopped or stopping
+	Stop();
+	// rest of the cleanup handled in Main.cpp
```

Comment:
There are two comments in the chunk you quoted. Which one do you mean?

![chunk](https://i.imgur.com/wUQjy4q.png)